### DISCUSSION TITLE
Linux installation

### DISCUSSION THREAD

**Discussion by ThiagoRBM** [freeplane/freeplane#1480] [Help]
Hello!
I'm trying to install freeplane on manjaro linux, but I'm quite noob and having a hard time doing it hehehe
I donwloaded this folder: freeplane_srcpure-1.11.7.tar.gz
But there is no "make" and the like inside it and I don't know what to do. Sorry for my "noob" question and any help will be much appreciated!
I also didn't find any topic about this here, sorry if I already exists.

**Comment by ThiagoRBM** [comment]
I just downloaded the "freeplane_bin-1.11.7.zip" and run the "freeplane.sh". Not sure if it is the best way, but it is running :)

**Reply by euu2021** [reply]
I use this Portable version on Windows, and it works fine, just like the installer version.
